package.json

```diff
@@ -125,16 +123,12 @@
         "title": "Arduino: Select Serial Port"
       },
       {
-        "command": "arduino.changeBaudRate",
-        "title": "Arduino: Change Baud Rate"
+        "command": "arduino.openSerialMonitor",
+        "title": "Arduino: Open Serial Monitor"
       },
       {
         "command": "arduino.changeTimestampFormat",
-        "title": "Arduino: Change Timestamp Format"
-      },
-      {
-        "command": "arduino.openSerialMonitor",
-        "title": "Arduino: Open Serial Monitor"
+        "title": "Arduino: Change Timestamp"
```

Comment:
Why was this title changed?

Reply:
I changed it and then changed back, I forgot the last part of the string, I'll change it.